library/tiqr/Tiqr/Message/GCM.php

```diff
@@ -24,9 +24,15 @@
 require_once 'Zend/Mobile/Push/Gcm.php';
 require_once 'Zend/Mobile/Push/Message/Gcm.php';
 
+require_once 'Tiqr/Message/Firebase.php';
+
 /**
  * Android Cloud To Device Messaging message.
  * @author peter
+ *
+ * @WARN This is a locally managed file. This implementation allows for a fallback GCM send event.
```

Comment:
Is this still a locally managed file? Isn't the whole point of this PR that we create a more robust temporary solution for this problem?

Reply:
Remove the comment about the 'locally managed' file